src/ImageRunModal.jsx

```diff
@@ -258,7 +258,6 @@ export class ImageRunModal extends React.Component {
                 .then(reply => varlink.call(utils.PODMAN_ADDRESS, "io.podman.StartContainer", { name: reply.container }))
                 .then(() => {
```

Comment:
Same here, could now be `then(this.props.close)`

Reply:
done